WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -77,25 +102,32 @@ class PostListFragment : Fragment(),
     private var actionableEmptyView: ActionableEmptyView? = null
     private var progressLoadMore: ProgressBar? = null
 
-    private var canLoadMorePosts = true
     private var targetPost: PostModel? = null
-    private var isFetchingPosts: Boolean = false
     private var shouldCancelPendingDraftNotification = false
     private var postIdForPostToBeDeleted = 0
 
-    private val trashedPosts = ArrayList<PostModel>()
+    private val uploadedPostRemoteIds = ArrayList<Long>()
+    private val trashedPostIds = ArrayList<Pair<Int, Long>>()
 
     private lateinit var nonNullActivity: Activity
     private lateinit var site: SiteModel
 
     @Inject internal lateinit var siteStore: SiteStore
     @Inject internal lateinit var postStore: PostStore
+    @Inject internal lateinit var listStore: ListStore
     @Inject internal lateinit var dispatcher: Dispatcher
 
+    private var listManager: ListManager<PostModel>? = null
+    private var refreshListDataJob: Job? = null
+    private val listDescriptor: PostListDescriptor by lazy {
```

Comment:
If you want to try different filters as mentioned in the PR, this is the value you need to modify.